src/tests/components/lists/FamilyList.test.tsx

```diff
@@ -85,13 +73,31 @@ describe('FamilyList', () => {
       const allFamilies = screen.getAllByText(/Family/)
 
       const indexFamilyOne = allFamilies.findIndex(
-        (element) => element.textContent === 'Family One',
+        (element) => element.textContent === mockFamiliesData[1].title,
       )
       const indexFamilyTwo = allFamilies.findIndex(
-        (element) => element.textContent === 'Family Two',
+        (element) => element.textContent === mockFamiliesData[0].title,
       )
 
       expect(indexFamilyOne).toBeGreaterThan(indexFamilyTwo)
     })
   })
+
+  it('shows a warning icon only for families without documents or events', async () => {
+    const familyIdWithoutDocumentsOrEvents = mockFamiliesData[2].import_id
```

Comment:
This is particularly difficult to follow given we are using array indexing and have to refer to the other mock file to associate each index with each mocked object. Could I suggest you define the mock objects at the top of this file, something like `const UNFCCCFamily = mockFamiliesData[0]` and then refer to it as `UNFCCCFamily` within this test file?

Reply:
Yes, it is a good idea for make it more explicit